File: JAVA/1347. Minimum Number of Steps to Make Two Strings Anagram.java
class Solution {
    public int minSteps(String s, String t) {
        int[] a=new int[26];
        for(int i=0;i<s.length();i++){
            a[t.charAt(i)-'a']++;
            a[s.charAt(i)-'a']--;
        }
        int c=0;
        for(int i=0;i<26;i++)
        c+=Math.max(0,a[i]);
        return c;
    }
}

File: C++/1347. Minimum Number of Steps to Make Two Strings Anagram.cpp
class Solution {
public:
    int minSteps(string s, string t) {
        vector<int>a(26,0);
        int c=0;
        for(int i=0;i<s.size();i++){
            a[t[i]-'a']++;
            a[s[i]-'a']--;
        }
        for(int i=0;i<26;i++)
        c+=max(0,a[i]);
        return c;
    }
};